qiskit_nature/algorithms/pes_samplers/__init__.py

```diff
@@ -71,6 +71,15 @@
     VibrationalStructureBase,
 )
 from .bopes_sampler import BOPESSampler
+from ...deprecation import warn_deprecated, DeprecatedType, NatureDeprecationWarning
+
+warn_deprecated(
+    "0.5.0",
+    old_type=DeprecatedType.PACKAGE,
+    old_name="qiskit_nature.algorithms.pes_sampler",
```

Comment:
Should we refer to #750 to some degree here? It might result in at least the `extrapolators` going somewhere else..